<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThemeDialog</class>
 <widget class="QDialog" name="ThemeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Theme Editor</string>
  </property>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="redBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>95</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="greenBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>125</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="blueBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>155</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Red:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Green:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue:</string>
   </property>
  </widget>
  <widget class="QSlider" name="blueSlider">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="greenSlider">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="redSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>250</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>1</y>
     <width>57</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>5</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit Elements</string>
   </property>
   <widget class="QRadioButton" name="darkTilesButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Dark Tiles</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="lightTilesButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Light Tiles</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="darkPiecesButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Dark Pieces</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="lightPiecesButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Light Pieces</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>130</y>
      <width>91</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>Cursor Hover:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="darkPiecesHoverButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Dark Pieces</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="lightPiecesHoverButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Light Pieces</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tileHighlightButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tile Highlight</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="attackingButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Attacking Highlight</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
